src/Chapter1.hs

```diff
@@ -490,8 +490,11 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
-
+lastDigit :: Int -> Int
+lastDigit n
+  | n >=0 = mod n 10
+  | (mod n 10) == 0 = 0
+  | otherwise = 10 - (mod n 10)
```

Comment:
That's a correction solution 👍🏻 And it can be improved even further by simplifying some of your clauses. You can use the standard function `abs` which returns the absolute value of a number. And it will help you to handle both positive and negative numbers easily 🙂

Reply:
Oh yes, this is super elegant, instead of having to treat both cases separately I can combine it into one case which is shorter and easier to deal with. Thanks!